CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## [3.11]
+- Add support for Finshark (Open Banking).
```

Comment:
```suggestion
- Add support for Open Banking (Using Finshark)
```

Our payment method is Open Banking. The fact that we use Finshark as engine is just for more info to the merchants.